File: ConnectFour/Python/ConnectFour/ConnectFour.py
import os
import random
import time
import View

from abc import ABCMeta, abstractmethod

class connectFour(object): 
    def main():
     #   //
      #  // --- Create the Model, View and Controller
       # //
       controller = controller()
       view = View(controller)
       model = Model(view)
       controller.setup(model);
      #  //
       # // Show view
       # //
       view.setVisible(true)
       

File: ConnectFour/src/connectFour.java
public class connectFour {
	   public static void main(String[] args)
	   {
      //
      // --- Create the Model, View and Controller
      //
       Controller controller = new Controller();
       View view = new View(controller);
       Model model = new Model(view);
       controller.setup(model);
       //
       // Show view
       //
       view.setVisible(true);
   }
}
